Flutter/calendar/drag-drop.md

```diff
@@ -16,7 +16,7 @@ Easily reschedule an appointment by dragging it from one time slot or month cell
 To perform drag-and-drop operations within the calendar, enable the [allowDragAndDrop](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/SfCalendar/allowDragAndDrop.html) property of SfCalendar.
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="9" %}
```

Comment:
Highlight below topics

https://help.syncfusion.com/flutter/calendar/timezone#create-appointment-in-different-time-zones
https://help.syncfusion.com/flutter/calendar/timezone#display-appointments-based-on-calendar-time-zone

Reply:
Timezone highlighted.

Reply:
Not hihglighted
https://help.syncfusion.com/flutter/calendar/timezone#create-appointment-in-different-time-zones

Reply:
timezone highlighted.